### ISSUE TITLE
how to run?

### ISSUE THREAD

**Issue by helojo** [rangle/rangle-starter#200]
I cloned this project,and run npm run test or npm install,the console allways scroll  "Resolved "redux"" text. how can i do make it run success?

**Comment by helojo** [comment]
any one is here?

**Comment by troyt-42** [comment]
@helojo sorry for the late reply, and like said in the README.md:

> Currently, we maintain starters for the following tech stacks:
> [Angular 2 with TypeScript](https://github.com/rangle/angular2-starter)
> [React with ES6](https://github.com/rangle/react-starter)
> [Angular 1 with TypeScript and Redux](https://github.com/rangle/angular-redux-starter)
> [API Koa Starter](https://github.com/rangle/api-koa-starter)

You can clone one of these repo and run npm install there.

**Comment by jphustman** [comment]
You have to run:

``` bash
$ rangle-starter
```

which will create a project, then in that project do

``` bash
$ cd myProject
$ npm install
$ npm run dev
```

**Comment by helojo** [comment]
thanks for up answers,i was clone https://github.com/rangle/typescript-react-redux-example.git,and then,run "npm install" the console allways scroll Resolved "redux " or other info. i do not know why this counld not init the inviriment.

**Comment by jphustman** [comment]
Oh I see, if you are still having problems can you post the npm-debug.log (or the output you are getting in the console)?  If it's really long maybe paste it in http://pastebin.com/

**Comment by helojo** [comment]
WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:20:1: Failed to find 'basscss-type-scale'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:21:1: Failed to find 'basscss-typography'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:22:1: Failed to find 'basscss-layout'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:23:1: Failed to find 'basscss-align'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:24:1: Failed to find 'basscss-margin'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:25:1: Failed to find 'basscss-padding'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:26:1: Failed to find 'basscss-grid'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:27:1: Failed to find 'basscss-flexbox'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:28:1: Failed to find 'basscss-position'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:29:1: Failed to find 'basscss-border'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

WARNING in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
postcss-import: E:\workspace\typescript\typescript-react-redux-example\src\style
s\index.css:30:1: Failed to find 'basscss-hide'
    in [
        E:\workspace\typescript\typescript-react-redux-example\src\styles
    ]

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-type-sca
le in E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 3:10-94

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-typograp
hy in E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 4:10-94

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-layout i
n E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 5:10-90

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-align in
 E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 6:10-89

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-margin i
n E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 7:10-90

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-padding
in E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 8:10-91

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-grid in
E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 9:10-88

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-flexbox
in E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 10:10-91

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-position
 in E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 11:10-92

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-border i
n E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 12:10-90

ERROR in ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./basscss-hide in
E:\workspace\typescript\typescript-react-redux-example\src\styles
 @ ./~/css-loader?-minimize!./~/postcss-loader!./src/styles/index.css 13:10-88
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 389 bytes [rendered]
        [0] ./~/html-webpack-plugin/lib/loader.js!./src/index.html 389 bytes {0}
 [built]

**Comment by helojo** [comment]
up text is i run commond "npm run dev" in console.  react redux-thunk and others i was installed at first time

**Comment by rbrander** [comment]
@helojo This is an old issue and appears to be have been resolved in issue #216. Since this issue is older with the same error, I'm going to assume this issue is now obsolete as #216 is newer and is closed.  Closing this issue.
